src/blog/lagom-1-3-7.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Lagom 1.3.7 is released!
+date: 2017-08-18
+author_github: rcavalcanti
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.3.7
+---
+
+The Lagom team has released Lagom 1.3.7. For a list of the specific changes in Lagom 1.3.7, see the [change log](/changelog.html).
+
+As an overview, this release contains the following:
+
+- Update Akka to 2.4.20 - (Security Patch Release)[http://akka.io/blog/news/2017/08/10/akka-2.4.20-released] 
+- Changed logging default settings - file appender removed (see details below)
+- Experimental Service Gateway based on akka-http
+- Storing read-side processor offsets for unhandled events
+- Upgrade to latest sbteclipse
+- Automatic load of `LogbackLoggerConfigurator` for Lagom Scala Applications
+- Other small bug fixes and minor enhancements
+
+We decided to remove the existing file appender. A microservices based application must keep its services movable. In that case, it doesn’t make sense to write to files. Instead, your orchestration solution of choice would better capture the _stdout_ and aggregate it. If desired, it can be explicitly added as explained in the [Java Guide](https://www.lagomframework.com/documentation/1.3.x/java/SettingsLogger.html#Configuring-Logging) and [Scala Guide](https://www.lagomframework.com/documentation/1.3.x/scala/SettingsLogger.html#Configuring-Logging)
```

Comment:
I would put a heading here, since it might not be immediately obvious that you're talking about logging here.

Also, this only mentions one of the changes to logging, and there are several. In particular, people might have configured log parsers based on the previous layout. As I mentioned before, I think it would be helpful if we could instruct people on how exactly to retain the previous format if desired (i.e., by linking to the 1.3.6-tagged file in GitHub and telling them where to put it in their services).